source/Nevermore/Util/DataModificationQueryBuilder.cs

```diff
@@ -494,7 +491,7 @@ class RelatedDocumentTableData
             public string SchemaName { get; set; }
             public string IdColumnName { get; set; }
             public string RelatedDocumentIdColumnName { get; set; }
-            public (string parentIdVariable, string relatedDocumentId, string relatedTableName)[] Related { get; set; }
+            public (string parentId, string relatedDocumentId, string relatedTableName)[] Related { get; set; }
```

Comment:
I need the actual value of the FK record, not its variable to insert it into the table valued parameter